experimental/language-generation/docs/structured-response-template.md

```diff
@@ -0,0 +1,374 @@
+# Structured response template
+Structured response template enable you to define a complex structure that supports all the goodness of LG (templating, composition, substitution) while leaving the interpretation of the structured response up to the caller of the LG library. 
+
+For bot applications, we will natively support ability to - 
+- activity definition
+- card definition
+- any [chatdown][1] style constructs
+
+[Bot Framework activity][2] includes several fields that are of interest and might require the abilty for the user to customize or control but to start with the following are most widely used properties that should be configurable via an Activity template definition - 
+
+| Property          | Use case                                                                                                                          |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| Text              | Display text used by the channel to render visually                                                                               |
+| Speak             | Spoken text used by the channel to render audibly                                                                                 |
```

Comment:
if I don't define speak will Text be used instead? 
If I dont define Text will Speak be used instead? 
What is the relationship between the different properties?

Reply:
Clarified.